includes/class-amp-enhancements.php

```diff
@@ -47,16 +59,16 @@ public static function should_use_amp_plus( $context = null ) {
 	}
 
 	/**
-	 * Allow certain scripts to be included in AMP pages.
+	 * Sanitize AMP Plugin errors to reject some errors.
 	 *
-	 * @param array $sanitizers The array of sanitizers, 'MyClassName' => [] // array of constructor params for class.
+	 * @param bool|null $is_sanitized If null, the error will be handled. If false, rejected.
+	 * @param object    $error The AMP sanitisation error.
 	 */
-	public static function amp_content_sanitizers( $sanitizers ) {
-		if ( self::should_use_amp_plus( 'gam' ) ) {
-			require_once NEWSPACK_ABSPATH . 'includes/amp-sanitizers/class-amp-sanitizer-gam.php';
-			$sanitizers['AMP_Sanitizer_GAM'] = [];
+	public static function amp_validation_error_sanitized( $is_sanitized, $error ) {
+		if ( isset( $error['node_attributes'], $error['node_attributes']['amp-plus-allowed'] ) ) {
```

Comment:
To avoid extra validation errors from being emitted by the presence of this attribute, you may want to use a `data-` attribute instead.

Reply:
Done in 3a1ee29bd5b6198c05e0282ffd7a69af681fc770 🙌